usr/share/rear/rescue/GNU/Linux/310_network_devices.sh

```diff
@@ -403,11 +408,19 @@ function ipaddr_setup () {
     if [ -n "$ipaddrs" ] ; then
         # If some IP is found for the network interface, then use them
         for ipaddr in $ipaddrs ; do
+            if IsInArray "${ipaddr%%/*}" "${EXCLUDE_IP_ADDRESSES[@]}"; then
+                LogPrint "Excluding IP address '$ipaddr' per EXCLUDE_IP_ADDRESSES directive even through it's defined in mapping file '$CONFIG_DIR/mappings/ip_addresses'."
```

Comment:
Does this situation indicate that something is wrong? I think not, because `EXCLUDE_IP_ADDRESSES` are intended for the rescue image only, while the mappings get applied to the recovered system.

Reply:
You can create your own mapping file during creation of the rescue, but in real life I never saw such thing.

Reply:
The mapping files are not well documented (I found only some examples in the docs directory), maybe that's why

Reply:
My personal gut feeling is that in real life
admins don't need such migration in case of disaster recovery
because they want to get the same system back in the same environment.
I think when they like to move a system into a different (networking) environment
they use something else than ReaR.
I think the only migration that admins need in real life during disaster recovery
is disk layout migration i.e. when the new disks are a bit different than the old ones.
But I think even for disk layout migration no miracles are expected.
I assume no experienced admin expects that he could "just recover" a server
that died on a spinning SATA disk onto new hardware with an NVMe SSD, cf.
https://github.com/rear/rear/issues/2713#issuecomment-1003965554